pkg/specs/pgbouncer/deployments.go

```diff
@@ -150,6 +152,20 @@ func Deployment(pooler *apiv1.Pooler, cluster *apiv1.Cluster) (*appsv1.Deploymen
 	}, nil
 }
 
+func computeTemplateHash(pooler *apiv1.Pooler, operatorImageName string) (string, error) {
+	type deploymentHash struct {
```

Comment:
The object used to calculate the hash is changed, so the pgbouncer pod will be restarted after upgrading the operator

Reply:
Yes, agreed